_docs/reference/labels.md

```diff
@@ -29,6 +31,37 @@ To create a volume with a feature labels:
 storageos volume create --label storageos.com/throttle=true --label storageos.com/replicas=1 
 ```
 
+## StorageOS Pool labels
+
+Pools do not have any labels present on them by default.
+
+| Feature             | Label                               | Values                               | Description                                                                                                                                                                                                                        |
+| :------------------ | :---------------------------------- | :----------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------                                                                                     |
+| Overcommitment      | `storageos.com/overcommit`          | integers [+]                         | Sets the percentage of overcommitment allowed for the pool.                                                                                                                                                                        |
+
+To add overcommit labels to a pool: 
+
+```bash
+storageos pool update --label-add storageos.com/overcommit=20 default
+```
+This would result in the total pool size being increased by 20% and the
+maximum volume size per node being increased by 20%.
+
+If you have 120GB pool comprised of 3 nodes with 40GB disks then the maximum
```

Comment:
Can we change tense to 3rd person? 'Given a 120GB pool...' - reads slightly more professionally I think.